ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/MSVS_2010/tasks/main.yml

```diff
@@ -20,6 +20,10 @@
   when: (vs2010_installed.stat.exists == false)
   tags: MSVS_2010
 
+- name: Register Visual Studio 2010 DIA SDK shared libraries
+  raw: 'for /r "C:\Program Files (x86)\Microsoft Visual Studio 10.0\DIA SDK" %F in ( msdia*.dll ) do regsvr32 /s "%~F"'
+  tags: MSVS_2010
+
```

Comment:
How will this impact a machine that also has Microsoft Visual Studio 12.0 installed?  That has its own version of DIA SDK, should those also be registered?  Or would this cause conflict for the same machine that uses both toolchains?

Reply:
VS12 has a distinct CLSID for the 'Debug Information Accessor' implementation so it won't interfere with the one for VS10.

Unfortunately, VS12 doesn't register its implementation so this PR will need to be amended.